binderhub/app.py

```diff
@@ -207,6 +207,14 @@ def _valid_badge_base_url(self, proposal):
         config=True,
     )
 
+    registry_class = Type(
+        DockerRegistry,
+        help="""
+        Registry class implementation, change to define your own
```

Comment:
```suggestion
        Change this to use different docker container registries.
        The default works with GCR, ACR and DockerHub. Use `AWSElasticContainerRegistry` on AWS.
```

How about something like this?

Reply:
Updated it in that direction.